test/cpp/end2end/client_lb_end2end_test.cc

```diff
@@ -353,6 +353,23 @@ TEST_F(ClientLbEnd2endTest, PickFirst) {
   EXPECT_EQ("pick_first", channel->GetLoadBalancingPolicyName());
 }
 
+TEST_F(ClientLbEnd2endTest, PickFirstProcessPending) {
+  StartServers(1);                  // Single server
+  auto channel = BuildChannel("");  // test that pick first is the default.
```

Comment:
We should use "pick_first" here and below. The LB policy default choice is tested in PickFirst above. Other tests in this file should specify the LB policy to use.

Reply:
Yeah, I was being consistent with the way we do it in other tests. I suggest we do that on its own separate (and trivial) PR.